test/unit-tests/components/views/settings/__snapshots__/Notifications-test.tsx.snap

```diff
@@ -2,94 +2,114 @@
+          Turn off to disable notifications on all your devices and sessions
         </span>
-      </label>
-      <div
-        aria-checked="true"
-        aria-disabled="false"
-        aria-label="Show all activity in the room list (dots or number of unread messages)"
-        class="mx_AccessibleButton mx_ToggleSwitch mx_ToggleSwitch_on mx_ToggleSwitch_enabled"
-        id="mx_SettingsFlag_testid_0"
-        role="switch"
-        tabindex="0"
-      >
-        <div
-          class="mx_ToggleSwitch_ball"
-        />
       </div>
     </div>
-    <div
-      class="mx_SettingsFlag"
+    <form
+      class="_root_19upo_16"
     >
-      <label
-        class="mx_SettingsFlag_label"
-        for="mx_SettingsFlag_testid_1"
+      <div
+        class="_inline-field_19upo_32"
       >
-        <span
-          class="mx_SettingsFlag_labelText"
+        <div
+          class="_inline-field-control_19upo_44"
         >
-          Only show notifications in the thread activity centre
-        </span>
-      </label>
+          <div
+            class="_container_19o42_10"
+          >
+            <input
+              checked=""
+              class="_input_19o42_24"
+              id="mx_SettingsFlag_testid_0"
+              role="switch"
+              type="checkbox"
+            />
+            <div
+              class="_ui_19o42_34"
+            />
+          </div>
+        </div>
+        <div
+          class="_inline-field-body_19upo_38"
+        >
+          <label
+            class="_label_19upo_59"
+            for="mx_SettingsFlag_testid_0"
+          >
+            Show all activity in the room list (dots or number of unread messages)
+          </label>
+        </div>
+      </div>
       <div
-        aria-checked="true"
-        aria-disabled="false"
-        aria-label="Only show notifications in the thread activity centre"
-        class="mx_AccessibleButton mx_ToggleSwitch mx_ToggleSwitch_on mx_ToggleSwitch_enabled"
-        id="mx_SettingsFlag_testid_1"
-        role="switch"
-        tabindex="0"
+        class="_inline-field_19upo_32"
       >
         <div
-          class="mx_ToggleSwitch_ball"
-        />
+          class="_inline-field-control_19upo_44"
+        >
+          <div
+            class="_container_19o42_10"
+          >
+            <input
+              checked=""
+              class="_input_19o42_24"
+              id="mx_SettingsFlag_testid_1"
+              role="switch"
+              type="checkbox"
+            />
+            <div
+              class="_ui_19o42_34"
+            />
+          </div>
+        </div>
+        <div
+          class="_inline-field-body_19upo_38"
+        >
+          <label
+            class="_label_19upo_59"
+            for="mx_SettingsFlag_testid_1"
+          >
+            Only show notifications in the thread activity centre
+          </label>
+        </div>
       </div>
-    </div>
-  </div>
+    </form>
+  </form>
```

Comment:
A quick google suggests nesting forms is not ok https://www.w3.org/TR/html5/forms.html#the-form-element

> Flow content, but with no form element descendants.